.github/workflows/build.yml

```
@@ -122,151 +80,20 @@ jobs:
-      id: env
-    - name: Upload ${{ matrix.name }} Release Asset
-      uses: actions/upload-release-asset@v1.0.1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.env.outputs.upload_url }}
-        asset_path: webrtc.${{ matrix.name }}.tar.gz/webrtc.tar.gz
-        asset_name: webrtc.${{ matrix.name }}.tar.gz
-        asset_content_type: application/gzip
-  upload-assets-apple:
-    strategy:
-      fail-fast: false
-      matrix:
-        name:
-          - WebRTC.xcframework
-          - LiveKitWebRTC.xcframework
-    name: Release ${{ matrix.name }}
-    if: contains(github.ref, 'tags/m')
-    needs:
-      - create-release
-    runs-on: ubuntu-latest
-    steps:
-    - name: Download ${{ matrix.name }}
-      uses: actions/download-artifact@v4
-      with:
-        name: ${{ matrix.name }}
-    - uses: actions/download-artifact@v4
-      with:
-        name: create-release.env
-    - name: Env to output
-      shell: bash
-      run: |
-        ls -R
-        source create-release.env
-        echo "upload_url=$UPLOAD_URL" >> "$GITHUB_OUTPUT"
-      id: env
-    - name: Upload ${{ matrix.name }} Release Asset
-      uses: actions/upload-release-asset@v1.0.1
+        path: artifacts
+    - uses: softprops/action-gh-release@v2
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
-        upload_url: ${{ steps.env.outputs.upload_url }}
-        asset_path: ${{ matrix.name }}.zip
-        asset_name: ${{ matrix.name }}
-        asset_content_type: application/zip
-  # Windows だけ Content-Type が違うので別で記述する
-  # Windows has a different Content-Type, so handle it separately.
-  # upload-assets-windows:
-  #   strategy:
-  #     fail-fast: false
-  #     matrix:
-  #       name:
-  #         - windows_x86_64
-  #         - windows_arm64
-  #   name: Upload windows
-  #   if: contains(github.ref, 'tags/m')
-  #   needs:
-  #     - create-release
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #   - name: Checkout code
-  #     uses: actions/checkout@v4
-  #   - name: Download ${{ matrix.name }}
-  #     uses: actions/download-artifact@v4
-  #     with:
-  #       name: webrtc.${{ matrix.name }}.zip
-  #   - uses: actions/download-artifact@v4
-  #     with:
-  #       name: create-release.env
-  #   - name: Env to output
-  #     shell: bash
-  #     run: |
-  #       source create-release.env/create-release.env
-  #       echo "upload_url=$UPLOAD_URL" >> "$GITHUB_OUTPUT"
-  #     id: env
-  #   - name: Upload windows Release Asset
-  #     uses: actions/upload-release-asset@v1.0.1
-  #     env:
-  #       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  #     with:
-  #       upload_url: ${{ steps.env.outputs.upload_url }}
-  #       asset_path: webrtc.${{ matrix.name }}.zip/webrtc.zip
-  #       asset_name: webrtc.${{ matrix.name }}.zip
-  #       asset_content_type: application/zip
+        files: |
+          artifacts/**/*.xcframework.zip
```

```suggestion
          artifacts/**/*.xcframework
```
Looks like they get uploaded as `*.xcframework` instead of `*.xcframework.zip`?

https://github.com/webrtc-sdk/webrtc-build/blob/381d8a5834178c187d1814f3eb6fe3e5b75a9cd4/.github/workflows/build.yml#L50